modules/account-lib/src/coin/hbar/utils.ts

```diff
@@ -2,10 +2,41 @@ import * as _ from 'lodash';
 import { Ed25519PublicKey, TransactionId, AccountId } from '@hashgraph/sdk';
 import BigNumber from 'bignumber.js';
 import * as hex from '@stablelib/hex';
+import { NetworkType } from '@bitgo/statics';
 import { AccountID, Timestamp } from './ifaces';
 
 const MAX_TINYBARS_AMOUNT = new BigNumber(2).pow(63).minus(1);
 
+const mainnetNodes: string[] = [
```

Comment:
can you reference where we pull these from?

Reply:
Yeah sure those are extracted from the hedera SDK index-node client. Maybe I can add a comment explicitly saying where we got them from.